repository-data/webfiles/src/main/resources/site/src/scss/components/_inset-text.scss

```diff
@@ -0,0 +1,31 @@
+.inset-text {
+    padding: 5px 0 5px $gutter;
+    border-left: 7px solid $blue;
+
+    &__title {
+        @include bold-font(24);
+    }
+
+    &__blocks {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+    }
+
+    &__block {
+        margin-top: 20px;
+        padding: 0;
+
+        &:first-child {
+            margin-top: 10px;
+        }
+
+        &-title {
+            @include core-font(18);
+        }
+
+        &-content {
+            margin-top: $gutter * 0.5;
+        }
+    }
+}
```

Comment:
Should this be added/from the UI toolkit?

Reply:
I am afraid our knowledge of the UI is very limited and tackling this may delay the release. Is it ok if we do not fix this now?